### ISSUE TITLE
BigNumberMath lib, whre to find?

### ISSUE THREAD

**Issue by tarasinf** [nickgammon/BigNumber#9]
@nickgammon Hi, I found the lib "BigNumberMath.h" that is mentioned that you are the author. Where I can find the source of code?

Thanks.

P.S. I came from [this](https://www.cerebralmeltdown.com/using-the-arduino-sun-tracking-heliostat-program/) solar tracking system.

**Comment by nickgammon** [comment]
Here on GitHub: https://github.com/nickgammon/BigNumber/tree/master/src/BigNumber

Or are you asking something else?

**Comment by tarasinf** [comment]
@nickgammon Yes, probably it's a bit different things. It's the lib that operate with your bigNumber lib and provide functions like sin, arcsin, cos, ... It calls "BigNumberMath.h" in that code.

**Comment by nickgammon** [comment]
I can't see any reference to BigNumberMath.h in my library. Where are you seeing that?

**Comment by tarasinf** [comment]
It had use together in that link. I know that it is not your responsibility, but I had the hope, that maybe you know something.

Don't care if you do not know. I just ask, if you know somethink about.

Thanks.

**Comment by nickgammon** [comment]
Ah I see you mean this:

```c++
	Adapted from:
	- libmath.b
	bc-1.06 
	13th September 2000
	
	- BigNumberMathFunctions.ino
	Mega_SunTracker_Heliostat_Program_V097_b @ cerebralmeltdown.com
	5th July 2012
	
	Original Authors: Philip A. Nelson & Gabriel
	Library Adaptation: sj
	v3
```

I don't know anything about that, sorry. I am not the author. The source code appears to be in BigNumberMath.cpp in the same folder as the .h file.

**Comment by tiarecorral** [comment]
> @nickgammon Hi, I found the lib "BigNumberMath.h" that is used with "BigNumber.h". Maybe someone knows something about this lib? Where I can find the source of code?
> 
> Thanks.
> 
> P.S. I came from [this](https://www.cerebralmeltdown.com/using-the-arduino-sun-tracking-heliostat-program/) solar tracking system.
Sorry! where did you find bignumbermath library?

**Comment by nickgammon** [comment]
I didn't find it anywhere. That is not my library, and I don't know where the other posters here found it.

The link posted above leads to [this page](https://www.cerebralmeltdown.com/arduino-sun-tracking-heliostat-program-download-page/). On that is [a link to their source code](https://www.cerebralmeltdown.com/wp-content/uploads//2012/12/SunTracking_Heliostat_Program_V098.zip).
